doc/releases/quincy.rst

```diff
@@ -5,6 +5,537 @@ Quincy
+* Add per OSD crush_device_class definition (`pr#50444 <https://github.com/ceph/ceph/pull/50444>`_, Francesco Pantano)
+* ceph-crash: drop privleges to run as "ceph" user, rather than root (CVE-2022-3650) (`pr#48805 <https://github.com/ceph/ceph/pull/48805>`_, Tim Serong, Guillaume Abrioux)
+* ceph-dencoder: Add erasure_code to denc-mod-osd's target_link_libraries (`pr#48028 <https://github.com/ceph/ceph/pull/48028>`_, Tim Serong)
+* ceph-exporter: cephadm changes (`pr#49771 <https://github.com/ceph/ceph/pull/49771>`_, Avan Thakkar)
+* ceph-mixing: fix ceph_hosts variable (`pr#48934 <https://github.com/ceph/ceph/pull/48934>`_, Tatjana Dehler)
+* ceph-volume/tests: add allowlist_externals to tox.ini (`pr#49788 <https://github.com/ceph/ceph/pull/49788>`_, Guillaume Abrioux)
+* ceph-volume/tests: fix lvm centos8-filestore-create job (`pr#48122 <https://github.com/ceph/ceph/pull/48122>`_, Guillaume Abrioux)
+* ceph-volume: add a retry in util.disk.remove_partition (`pr#47989 <https://github.com/ceph/ceph/pull/47989>`_, Guillaume Abrioux)
+* ceph-volume: do not raise RuntimeError in util.lsblk (`pr#50144 <https://github.com/ceph/ceph/pull/50144>`_, Guillaume Abrioux)
+* ceph-volume: fix a bug in get_all_devices_vgs() (`pr#49453 <https://github.com/ceph/ceph/pull/49453>`_, Guillaume Abrioux)
+* ceph-volume: fix a bug in lsblk_all() (`pr#49868 <https://github.com/ceph/ceph/pull/49868>`_, Guillaume Abrioux)
+* ceph-volume: fix regression in activate (`pr#48201 <https://github.com/ceph/ceph/pull/48201>`_, Guillaume Abrioux)
+* ceph-volume: legacy_encrypted() shouldn't call lsblk() when device is 'tmpfs' (`pr#50161 <https://github.com/ceph/ceph/pull/50161>`_, Guillaume Abrioux)
+* ceph.spec.in: disable system_pmdk on s390x for SUSE distros (`pr#48522 <https://github.com/ceph/ceph/pull/48522>`_, Tim Serong)
+* ceph.spec.in: Replace %usrmerged macro with regular version check (`pr#49831 <https://github.com/ceph/ceph/pull/49831>`_, Tim Serong)
+* ceph.spec.in: Use gcc11-c++ on openSUSE Leap 15.x (`pr#48058 <https://github.com/ceph/ceph/pull/48058>`_, Tim Serong)
+* ceph_fuse: retry the test_dentry_handling if fails (`pr#49942 <https://github.com/ceph/ceph/pull/49942>`_, Xiubo Li)
+* cephadm: add `ip_nonlocal_bind` to haproxy deployment (`pr#48211 <https://github.com/ceph/ceph/pull/48211>`_, Michael Fritch)
+* cephadm: Adding poststop actions and setting TimeoutStartSec to 200s (`pr#50447 <https://github.com/ceph/ceph/pull/50447>`_, Redouane Kachach)
+* cephadm: consider stdout to get container version (`pr#48208 <https://github.com/ceph/ceph/pull/48208>`_, Tatjana Dehler)
+* cephadm: don't overwrite cluster logrotate file (`pr#49849 <https://github.com/ceph/ceph/pull/49849>`_, Adam King)
+* cephadm: Fix disk size calculation (`pr#47945 <https://github.com/ceph/ceph/pull/47945>`_, Paul Cuzner)
+* cephadm: only pull host info from applied spec, don't try to parse yaml (`pr#49854 <https://github.com/ceph/ceph/pull/49854>`_, Adam King)
```

Comment:
Maybe this is too pedantic a comment for release notes, but the "only" in "only pull host info from applied spec, don't try to parse yaml" is in the maximally-ambiguous position. I suspect that what was intended was "pull host info only from applied spec, don't try to parse yaml", but it is possible that "pull only host info from applied spec, don't try to parse yaml" was intended.

Reply:
@zdover23 I think this change should have been done during PRs approval/merge process